app/src/main/res/layout/activity_contributor.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context="io.github.droidkaigi.confsched2018.presentation.contributor.ContributorsActivity"
+    >
+    <android.support.v4.widget.DrawerLayout
```

Comment:
Is this drawer necessary in contributors page?

Reply:
I'm sorry, I misunderstood it as spec. 🙇